week1/assignment/todo-tan/todo.rb

```diff
@@ -0,0 +1,167 @@
+require_relative "list"
+
+class Todo
+  def initialize(filename = "todo.md")
+    @filename = filename
+    @list = List.new("Today")
+    @trash_list = List.new("Trash")
+    @last_request = {}
+  end
+
+  def list
+    @list
+  end
```

Comment:
you already know this, so im just commenting that this could be replaced with `attr_reader :list` at the top of the class